gmso/lib/jsons/README.md

```diff
@@ -0,0 +1,30 @@
+# GMSO Potential Templates JSON Files
+
+This directory contains json representations of GMSO potential 
+templates. 
+
+A potential template object requires the following attributes:
+
+1. `name`: Name of the potential template
+2. `expression`: expression for the potential template
+3. `independent_variables`: Independent variables for the template 
+
+
+## For Developers
+To add/contribute a new `PotentialTemplate` there are the following ways:
```

Comment:
Minor comment, since there is only one way to contribute to the Potential Template now, I think we should fix this sentence. May be just have it as `To add/contribute a new 'PotentialTemplate':`

Reply:
agreed, but I think we can move that to a later pr. I think it is now in a merge-able state.